jib-plugins-common/src/test/java/com/google/cloud/tools/jib/api/HttpGetVerifier.java

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright 2018 Google LLC.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package com.google.cloud.tools.jib.api;
+
+import com.google.cloud.tools.jib.blob.Blobs;
+import java.io.IOException;
+import java.io.InputStream;
+import java.net.HttpURLConnection;
+import java.net.URL;
+import javax.annotation.Nullable;
+import org.junit.Assert;
+
+/** Verifies the response of HTTP GET. */
+public class HttpGetVerifier {
```

Comment:
nit: since this class is a little more than verifying the body now, perhaps we can rename this to something that encapsulates this new behavior?  Here are some patterns recommended in the java style guide for utility classes like this one:  `FooFactory`, `BarTester` or `BazClient`